src/backend/partaj/core/email.py

```diff
@@ -363,3 +363,42 @@ def send_validation_requested(cls, validation_request, activity):
         }
 
         cls.send(data)
+
+    @classmethod
+    def send_referral_closed(cls, contact, referral):
+        """
+        Send the "referral closed" email to the user who just closed the referral.
+        """
+
+        template_id = settings.SENDINBLUE["REFERRAL_CLOSED_TEMPLATE_ID"]
```

Comment:
This email should still include a link to the referral (even if just to let users quickly read the explanation for the closure)